devdocs/features.md

```diff
@@ -0,0 +1,16 @@
+| gRPC          |    All     |        1.0+ | All                                                                                |  Yes   |         Only in Go |                       For non go langagues, can't get method for long living connections, will mark method with `*` |
+| MySQL         |    All     |         All | All                                                                                |  Yes   |                 No |                                                                                                                 N/A |
+| PostgreSQL    |    All     |         All | All                                                                                |  Yes   |                 No |                                                                                                                 N/A |
+| Redis         |    All     |         All | All                                                                                |  Yes   |                 No | For already started connections, can't infer the number of the database, and won't add the `db.namespace` attribute |
+| MongoDB       |    All     |         >=5 | insert, update, find, delete, findAndModify, aggregate, count, distinct, mapReduce |  Yes   |                 No |                                                                                  no support for compressed payloads |
+| Kafka         |    All     |         All | produce, fetch                                                                     |  Yes   |                 No |         Might fail getting topic name for fetch requests in newer versions of kafka (where Fetch api version >= 13) |
+| JsonRPC       |     Go     |         All | -                                                                                  |  Yes   |                 No |                                                                                                                 N/A |
+| GraphQL       | All but Go |         All | All                                                                                |  Yes   |                 No |                                                                                                                 N/A |
+| Elasticsearch | All but Go |      > 7.14 | /_search                                                                           |  Yes   |                 No |                                                                                                                 N/A |
+| Aws S3        | All but Go |             | ?                                                                                  |  Yes   |                 No |                                                                                                                 N/A |
```

Comment:
```suggestion
| AWS S3        | All but Go |             | ?                                                                                  |  Yes   |                 No |                                                                                                                 N/A |
```
As of now we support basic crud operations such as `CreateBucket`, `DeleteBucket`, `PutObject`, `DeleteObject`, `ListBuckets`, `ListObjects`, `GetObject`